views-and-forms-java/src/test/java/views/and/forms/java/SurveyControllerTest.java

```diff
@@ -0,0 +1,72 @@
+package views.and.forms.java;
+
+import io.micronaut.test.annotation.MicronautTest;
+import org.junit.jupiter.api.Test;
+import views.and.forms.java.controllers.SurveyController;
+import views.and.forms.java.model.FormData;
+import views.and.forms.java.model.FruitChoices;
+
+import javax.inject.Inject;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@MicronautTest
+public class SurveyControllerTest {
+
+    @Inject
+    SurveyController itemUnderTest;
```

Comment:
We shouldn't be injecting the controller here to test anything. Either invoke the endpoint with something like Geb to test the HTML or move the logic into a service layer and inject that